recipes/qcelemental/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "qcelemental" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "32433771dd6b8e9c9723cf5525b3b8fba34b695cbc6e281d83f0f43f28e69dc5" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/MolSSI/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    noarch: python
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - numpy
+        - pint
+        - python
+
+test:
+    imports:
+        - qcelemental
+    requires:
+        - deepdiff
+        - pytest
+    commands:
+        - py.test --pyargs qcelemental
+
+about:
+    home: https://github.com/MolSSI/QCElemental
+    dev_url: https://github.com/MolSSI/QCElemental
+    doc_url: https://qcelemental.readthedocs.io/en/latest/
+    doc_source_url: https://github.com/MolSSI/QCElemental/blob/master/docs/source/index.rst
+    license: BSD-3-Clause
+    license_url: https://opensource.org/licenses/BSD-3-Clause
+    license_file: LICENSE
+    license_family: BSD
+    summary: QCElemental is a resource module for quantum chemistry containing physical constants and periodic table data from NIST and molecule handlers.
+    # description: >
+
+extra:
+    recipe-maintainers:
+        - loriab
```

Comment:
@loriab should respond here that they are OK being a maintainer.

Reply:
Affirmative.